### ISSUE TITLE
are you allowed to submit a new thing to add to the list with issues? if not, ignore this:

### ISSUE THREAD

**Issue by sI1vre** [methanoliver/awesome-renpy#3]
https://github.com/Elckarow/Expreviewer
ren'py 7/8 layeredimage previewer by my friend elckarow

https://github.com/Elckarow/Autofocus
also from my friend elckarow, a tool to automatically focus characters with darkening, slight size changes based on whos talking, etc.

https://github.com/ShadowXJaw724/Renpy_Shooter
implement a shooter minigame into ren'py

https://github.com/LuqueDaniel/vscode-language-renpy
that one extension that adds renpy syntax to vsc

https://github.com/Rubiksman78/MonikA.I
usually just for a ddlc mod addon, but i feel like this should be put in the code examples for its weirdly advanced usage of ai models in ren'py

basically, ddlc fans are shockingly good at renpy

**Comment by methanoliver** [comment]
Sure you are, gimme a moment. :)
